PencilCase/PencilCase/scripts/Pages/Retail.ts

```diff
@@ -1,12 +1,164 @@
 ﻿import { PageBase } from './PageBase';
 import { Navigator } from '../Navigator';
 import * as Consts from './Consts';
+import { Product } from '../Models/Product';
+import { ProductRepository } from '../Repositories/ProductRepository';
+import { OrderRepository } from '../Repositories/OrderRepository';
+import { Order } from '../Models/Order';
+import * as Utils from '../Utils';
+import { OrderTypes } from '../Models/Order';
 
 export class Retail extends PageBase {
```

Comment:
Just copied from Wholesale.ts and updated the code to make it work in the retail way.
Need to refactor the code to extract a base class for ImportProduct, Wholesale and Retail page.